backends/dpdk/dpdkArch.cpp

```diff
@@ -1692,12 +1692,23 @@ SplitP4TableCommon::create_match_table(const IR::P4Table *tbl) {
 
     auto actionCall = new IR::MethodCallExpression(new IR::PathExpression(actionName));
     actionsList.push_back(new IR::ActionListElement(actionCall));
-    actionsList.push_back(new IR::ActionListElement(tbl->getDefaultAction()));
+    auto default_action = tbl->getDefaultAction();
+    if (default_action) {
+        if (auto mc = default_action->to<IR::MethodCallExpression>()) {
+            // Ignore action params of default action by creating new MethodCallExpression
+            auto defAction = new IR::MethodCallExpression(mc->method->to<IR::PathExpression>());
+            actionsList.push_back(new IR::ActionListElement(defAction));
+        }
+    }
+
+    auto constDefAction = tbl->properties->getProperty("default_action");
```

Comment:
could this be null?

Reply:
Thanks for pointing out. I will add a check for that